projects/schematics/src/add-spartacus/index.ts

```diff
@@ -290,8 +301,6 @@ export function addSpartacus(options: SpartacusOptions): Rule {
         module: 'app',
```

Comment:
This whole section `ensureModuleExists` will generate an empty module file `app-routing.module.ts` and import it in the `app.module.ts` so you should remove it.

Reply:
nice catch